webapp/packages/plugin-data-viewer/src/DatabaseDataModel/IDatabaseDataModel.ts

```diff
@@ -16,12 +16,18 @@ export interface IRequestEventData<TOptions = any, TResult extends IDatabaseData
   model: IDatabaseDataModel<TOptions, TResult>;
 }
 
+/** Represents an interface for interacting with a database. It is used for managing and requesting data. */
 export interface IDatabaseDataModel<TOptions = any, TResult extends IDatabaseDataResult = IDatabaseDataResult> {
   readonly id: string;
   readonly name: string | null;
   readonly source: IDatabaseDataSource<TOptions, TResult>;
+  /** Holds metadata about a data request. */
   readonly requestInfo: IRequestInfo;
   readonly supportedDataFormats: ResultDataFormat[];
+  /**
+   * Represents the count of records loaded from the database per request.
+   * It can be updated using the setCountGain method.
+   */
   readonly countGain: number;
```

Comment:
Represents the value by which the number of loaded rows will be increased when loading the next data portion